[2025-05-08T13:26:46.542-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-08T13:26:46.546-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: request_operation.modoOperacion scheduled__2025-05-08T18:26:00+00:00 [queued]>
[2025-05-08T13:26:46.550-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: request_operation.modoOperacion scheduled__2025-05-08T18:26:00+00:00 [queued]>
[2025-05-08T13:26:46.550-0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-05-08T13:26:46.567-0500] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): modoOperacion> on 2025-05-08 18:26:00+00:00
[2025-05-08T13:26:46.572-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'request_operation', 'modoOperacion', 'scheduled__2025-05-08T18:26:00+00:00', '--job-id', '1738', '--raw', '--subdir', 'DAGS_FOLDER/operations.py', '--cfg-path', '/tmp/tmp5n72up4e']
[2025-05-08T13:26:46.573-0500] {standard_task_runner.py:105} INFO - Job 1738: Subtask modoOperacion
[2025-05-08T13:26:46.575-0500] {logging_mixin.py:190} WARNING - /home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=74211) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-08T13:26:46.575-0500] {standard_task_runner.py:72} INFO - Started process 74242 to run task
[2025-05-08T13:26:46.600-0500] {task_command.py:467} INFO - Running <TaskInstance: request_operation.modoOperacion scheduled__2025-05-08T18:26:00+00:00 [running]> on host DESKTOP-0R7MK50.
[2025-05-08T13:26:46.649-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='request_operation' AIRFLOW_CTX_TASK_ID='modoOperacion' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T18:26:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-08T18:26:00+00:00'
[2025-05-08T13:26:46.650-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-08T13:26:47.839-0500] {logging_mixin.py:190} WARNING - /home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/urllib3/connectionpool.py:1097 InsecureRequestWarning: Unverified HTTPS request is being made to host 'auth.iqoption.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
[2025-05-08T13:26:49.584-0500] {python.py:240} INFO - Done. Returned value was: Modo PRACTICE. Balance actual: 9913.1
[2025-05-08T13:26:49.602-0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-08T13:26:49.602-0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=request_operation, task_id=modoOperacion, run_id=scheduled__2025-05-08T18:26:00+00:00, execution_date=20250508T182600, start_date=20250508T182646, end_date=20250508T182649
[2025-05-08T13:26:49.664-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-08T13:26:49.672-0500] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-08T13:26:49.673-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
